### ISSUE TITLE
[Feature Request] An option to add local notes or text to a manga in the library.

### ISSUE THREAD

**Issue by themagicalyang** [inorichi/tachiyomi#2155]
**App version:** 8.4

**Android version:** 6

**Issue/Request:** An option to add notes or text. No syncing or cloudsaving, simple saved to application text/notes.

**Other details:** The way I see it is could be some kind of button on the manga page or another tab like the tracking tab on the manga page. A text box to write what I want. And it is saved in the JSON backup. Have some character limit for sane usage I guess.

**Comment by Efreak** [comment]
In #2662 / [comment](https://github.com/inorichi/tachiyomi/issues/2662#issuecomment-595043700), @hXtreme said: 

> I personally really like this feature and have asked for it previously but, this feature would need the underlying database structure to be updated, something that isn't very likely to happen anytime soon though.

I don't think this necessarily requires changing the database structure. Just make the description field itself editable, and don't update it when refreshing the entire library unless it's empty (if refreshing a single item manually and description changes, prompt to ask if it should be updated)

**Comment by monMusu** [comment]
Yep, a SNIPPET tool would be nice. Where you screenshot a portion with a marquee and save it with or without a note. Would be very useful if you want to remember important events that occurred in a Manga.

**Comment by hXtreme** [comment]
@monMusu for complex note taking such as adding and editing screenshots... It would be best to use another app specialised for notes.

**Comment by arkon** [comment]
If anyone's interested in implementing this, it'd ideally be stored within a `ComicInfo.xml` file (ComicRack format, see #4965) in the chapter download location.
